pom.xml

```diff
@@ -140,10 +140,13 @@
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>3.2.0</version>
           <configuration>
+            <detectJavaApiLink>false</detectJavaApiLink>
+            <doclint>-accessibility</doclint><!-- Avoid complaints about table@summary -->
```

Comment:
What complaints?

Reply:
Java 8 complains without the things removed because they're disallowed in HTML 5.